.gitignore

```diff
@@ -22,6 +22,7 @@ public/dist/
 uploads
 modules/users/client/img/profile/uploads
 config/env/local.js
+config/env/local-*.js
```

Comment:
@jloveland What about the existing local.example.js file that we do want to commit into the repo?